ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/config/PicoCliDownstreamTlsClientAuthOptions.java

```diff
@@ -18,20 +18,20 @@
 
 import picocli.CommandLine.Option;
 
-public class PicoCliTlsClientCertificateOptions implements PkcsStoreConfig {
+public class PicoCliDownstreamTlsClientAuthOptions implements PkcsStoreConfig {
```

Comment:
don't love this being called "ClientAuthOptions" - maybe ClientIdentityOptions?

Reply:
I've renamed it back to `PicoCliClientTlsCertificateOptions`